### DISCUSSION TITLE
Release 2.1.1(20230806)

### DISCUSSION THREAD

**Discussion by bao-cn** [Lunova-Studio/DarkMode2#20] [Q&A]
## 注意：此版本需要清除注册表项！

1. 修复切换颜色模式时的异常报错。
2. 临时关闭触摸键盘切换功能，恢复时间待定。
3. 隐藏开发者模式入口。

<hr /><em>This discussion was created from the release <a href='https://github.com/Melon-Studio/DarkMode2/releases/tag/2.1.1.20230806-Release'>Release 2.1.1(20230806)</a>.</em>

**Comment by Xing-Cn** [comment]
请问能不能添加一个设置系统光标的开关（平时用的并非是系统默认的光标。

**Reply by bao-cn** [reply]
确实，当初没考虑到这种情况，预计下个版本上线此功能。
